docs/source/tips.rst

```diff
@@ -141,10 +141,16 @@ You can also use :func:`chainer.using_config` to change the configuration.
         y = chainer.functions.relu(x)
     print(type(y.data))
 
-.. testoutput::
+.. code-block:: console
```

Comment:
How about `none`, which is more consistent with what would be generated by `testoutput`?

Reply:
OK, fixed.